reg/g/pcds/epics/modules/RELEASE_SITE

```diff
@@ -0,0 +1,17 @@
+BASE_MODULE_VERSION     = R3.14.12-0.4.0
```

Comment:
3.14 looks kind of odd as the definitive version, is that what we're using in prod?

Reply:
Edit: yep, this is the prod file. Strange.

Reply:
My guess is this is how it went:
`/reg/g/pcds/epics/` was R3.14 at the beginning,
`/reg/g/pcds/epics/modules` was obvious for R3.14 modules,
`/reg/g/pcds/epics/extensions` was obvious for R3.14 extensions,
then R3.15 came along years later and the following was set up:
`/reg/g/pcds/epics/base/(version)` became base for (version) and modules were placed in 
`/reg/g/pcds/epics/(version)/modules`.

The original 'modules' directory was just left as-is for back-compatibility, though (without giving it too much thought) it seems like moving and symlinking would have been preferable to keep directory structure more sane.

Browsing around a bit more, the directories there are in bad need of a cleanup. I suspect it just won't happen given that someone/something may be using it there.

I left-aligned all the paths above but it made it look like I was going for a poem or something. Oh well.